files/src/main/scala/io/iteratee/files/package.scala

```diff
@@ -1,5 +1,132 @@
 package io.iteratee
 
+import cats.{ Eval, MonadError }
+import io.iteratee.internal.Step
+import java.io.{
+  BufferedInputStream,
+  BufferedReader,
+  File,
+  FileInputStream,
+  FileReader,
+  InputStream,
+  InputStreamReader
+}
+import java.util.zip.{ ZipEntry, ZipFile }
+import scala.Predef.genericArrayOps
+import scala.collection.JavaConverters._
+import scala.util.{ Left, Right }
+
 package object files {
   final object xor extends XorFileModule
+
+  final def readLines[F[_]](file: File)(implicit
+    F: MonadError[F, Throwable],
+    C: EffectCapture[F]
```

Comment:
before we put these everywhere, can we describe one concrete case where this will be different from `F.catchNonFatal`?